src/components/markdown/examples/markdown-composite.tsx

```diff
@@ -0,0 +1,38 @@
+import { LimelInputFieldCustomEvent } from '@limetech/lime-elements';
+import { Component, Host, State, h } from '@stencil/core';
+
+/**
+ * Composite example with a markdown input field and a markdown renderer.
+ */
+@Component({
+    tag: 'limel-example-markdown-composite',
+    styleUrl: 'markdown-composite.scss',
+    shadow: true,
+})
+export class MarkdownRenderContentExample {
+    @State()
+    private markdown = '# Hello, world!\n\nThis is **markdown**!';
+
+    public render() {
+        return (
+            <Host>
+                <limel-input-field
+                    label="Markdown to render"
+                    type="textarea"
+                    value={this.markdown}
+                    onChange={this.handleMarkdownChange}
+                />
+                <fieldset>
+                    <legend>Rendered markdown</legend>
+                    <limel-markdown value={this.markdown} />
+                </fieldset>
+            </Host>
+        );
+    }
```

Comment:
I normally don't add `<Host>` like this, unless I need a tag or attribute on the host itself. I'd recommend just using an array instead.

```suggestion
import { LimelInputFieldCustomEvent } from '@limetech/lime-elements';
import { Component, State, h } from '@stencil/core';

/**
 * Composite example 
 * Test your markdown code and see what you get in return in real-time.
 */
@Component({
    tag: 'limel-example-markdown-composite',
    styleUrl: 'markdown-composite.scss',
    shadow: true,
})
export class MarkdownRenderContentExample {
    @State()
    private markdown = '# Hello, world!\n\nThis is **markdown**!';

    public render() {
        return [
              <limel-input-field
                  label="Markdown to render"
                  type="textarea"
                  value={this.markdown}
                  onChange={this.handleMarkdownChange}
              />
              <fieldset>
                  <legend>Rendered markdown</legend>
                  <limel-markdown value={this.markdown} />
              </fieldset>
          ];
    }
```